data.json

```diff
@@ -44,6 +44,15 @@
     "dateAdded": "2020-10-01T00:00:00.000Z",
     "tags": ["hacktoberfest", "stickers", "clothing"]
   },
+  {
+    "name": "Appwrite",
```

Comment:
Please merge this with the next one (Appwrite Hacktoberfest 2019) :smile:

Reply:
Hopefully this is what you meant :-)